lib/adapters/sql.ex

```diff
@@ -181,7 +181,7 @@ defmodule Apartmentex.Adapters.SQL do
   end
 
   defp query(repo, sql, params, outer_queue_time, mapper, opts) do
-    {pool_mod, pool, timeout} = repo.__pool__
+    {pool_mod, pool, timeout, _} = repo.__pool__
```

Comment:
Looks like an operation timeout was added in 1.1.0.  Won't matter for Apartmentex, so I'll get this merged, and I will put these changes into a new branch, then get it out to hex.  I'll end up with three branches- this for ecto 1.1.x, another for ecto 1.0.x, and then the master for ecto 2.0.x which I'll be working on.  Thanks for the help!